src/electron/process_manager.ts

```diff
@@ -299,6 +301,13 @@ class ChildProcessHelper {
       }
     };
 
+    if (process.env.OUTLINE_DEBUG === 'true') {
```

Comment:
I'm a not a fan of depending on globals deep in the code.
Could you add a setDebug(bool) to TunnelManager, SSLocal and Tun2socks and call that in the caller of the TunnelManager constructor?

Reply:
Done in 5de426e24775db28a6024b6fad7c4a66e6b62b51